app/test_split_integer.py

```diff
@@ -1,21 +1,67 @@
 from app.split_integer import split_integer
+import pytest
 
 
-def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+@pytest.mark.parametrize("value, number_of_parts, expected_result", [
+    (8, 1, [8]),
+    (6, 2, [3, 3]),
+    (17, 4, [4, 4, 4, 5]),
+    (32, 6, [5, 5, 5, 5, 6, 6])
+])
+def test_sum_of_the_parts_should_be_equal_to_value(value: int,
+                                                   number_of_parts: int,
+                                                   expected_result: list
+                                                   ) -> None:
+    assert value == sum(expected_result)
 
 
-def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+@pytest.mark.parametrize("value, number_of_parts, expected_result", [
+    (8, 1, [8]),
+    (6, 2, [3, 3]),
+    (17, 4, [4, 4, 4, 5]),
+    (32, 6, [5, 5, 5, 5, 6, 6])
+])
+def test_should_split_into_equal_parts_when_value_divisible_by_parts(
+        value: int,
+        number_of_parts: int,
+        expected_result: list) -> None:
+    assert len(split_integer(value, number_of_parts)) == number_of_parts
+    assert max(expected_result) - min(expected_result) <= 1
 
 
-def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+@pytest.mark.parametrize("value, number_of_parts, expected_result", [
+    (8, 1, [8]),
+    (6, 2, [3, 3]),
```

Comment:
Test data should divide in one part - means second param must be 1 for this test